plugins/source/aws/codegen/recipes/servicequota.go

```diff
@@ -0,0 +1,57 @@
+package recipes
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/servicequotas/types"
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func ServiceQuotasResources() []*Resource {
+	resources := []*Resource{
+		{
+			SubService: "services",
+			Struct:     &types.ServiceInfo{},
+			SkipFields: []string{"ServiceCode", "ServiceName"},
+			Multiplex:  `client.ServiceAccountRegionMultiplexer("servicequotas")`,
+			ExtraColumns: append(
+				defaultRegionalColumnsPK,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "service_code",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("ServiceCode")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					}, {
+						Name:     "service_name",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("ServiceName")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
+			Relations: []string{
+				"Quotas()",
+			},
+		},
+		{
+			SubService: "quotas",
+			Struct:     &types.ServiceQuota{},
```

Comment:
```suggestion
			Struct:     &types.ServiceQuota{},
			```suggestion
			Struct:     &types.ServiceInfo{},
			Description: "https://docs.aws.amazon.com/servicequotas/2019-06-24/apireference/API_ServiceQuota.html",
```
```